go/vt/vtgate/evalengine/testcases/cases.go

```diff
@@ -1766,6 +1767,25 @@ func FnLastDay(yield Query) {
 	}
 }
 
+func FnFromDays(yield Query) {
+	for _, d := range inputConversions {
+		yield(fmt.Sprintf("FROM_DAYS(%s)", d), nil)
+	}
+
+	days := []string{
+		"0",
+		"1",
+		"366",
+		"365242",
+		"3652424",
+		"3652425",
```

Comment:
Generally we also try to include the cases from the MySQL docs. So here https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_from-days has `FROM_DAYS(730669)` as an example, so let's include that too.

Reply:
done.